typescript/.changeset/calm-ads-look.md

```diff
@@ -0,0 +1,5 @@
+---
+"@coinbase/cdp-sdk": minor
+---
+
+Added getUserOperation smart account action to TypeScript sdk
```

Comment:
No need to mention TypeScript SDK here, as this is the changelog for the TypeScript SDK
```suggestion
Added getUserOperation smart account action
```